src/Model.php

```diff
@@ -1249,6 +1275,9 @@ public function save($data = [], Persistence $to_persistence = null)
         }
 
         return $this->atomic(function () use ($to_persistence) {
+            if (($errors = $this->validate('save')) !== []) {
+                throw new ValidationException($errors);
```

Comment:
Probably you should add 2nd parameter here like this `throw new ValidationException($errors, 'save');`